packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -13,8 +13,8 @@ type UnitRowProps = {
   noHeaderValueCtaText?: string;
   children?: React.ReactNode;
   headerValueClassName?: string;
-  route?: string;
-  revealModal?: () => void;
+  route?: string | null;
+  revealModal?: (() => void) | null;
```

Comment:
Code clarity nitpick: I see this is potentially `null` because of the how the props are [composed](https://github.com/mozilla/fxa/pull/6187/files#diff-a313ff76f12b71cf55da0fc953b5dabbR76). In the spirit of keeping the intended type declaration clear (as in, `null` serves no purpose other than placeholder), could we maybe update either how the prop object is constructed (e.g. if the value is not present, don't set the property instead of setting it to `null`), **or** instead of `null` set the [alternative](https://github.com/mozilla/fxa/pull/6187/files#diff-a313ff76f12b71cf55da0fc953b5dabbR65) to `undefined` (that way the types can remain unchanged)?

Reply:
> or instead of null set the placeholder to undefined

Heh, I actually did this at first, but changed it to `null` since it usually feels weird to intentionally set something to `undefined`, although I know we do that in other places. I can change it back to `undefined` instead of `null` so the type looks a bit better.